pkg/devfile/parser/parse.go

```diff
@@ -61,7 +60,20 @@ func Parse(path string) (d DevfileObj, err error) {
 	if err != nil {
 		return d, err
 	}
-	return parseDevfile(d)
+	return parseDevfile(d, true)
+}
+
+// ParseRawDevfile populates the raw devfile data witout overring and merging
```

Comment:
without* & overriding*
probably also mention in more detail like no parent or plugin will be parsed

Reply:
Not sure if we even need this?

Reply:
We will need this, to update an existing devfile using writer will need to write to raw devfile content.